### DISCUSSION TITLE
Disable selection of blocks

### DISCUSSION THREAD

**Discussion by lofcz** [codex-team/editor.js#1893] [Q&A]
Is there a way to disable built-in rectangular selection of blocks?

![image](https://user-images.githubusercontent.com/10260230/143770733-cc18543a-4a99-4ab9-bae0-059bc2f2e1a6.png)

**Comment by gokhantaskan** [comment]
Any news about it?

**Comment by theophilhenry** [comment]
I also need this

**Comment by m-burger** [comment]
I also would like to configure the selection behavior (enabled/disabled).

**Comment by ShiXinClearBoth** [comment]
Is there any other way？
